linter.py

```diff
@@ -277,6 +302,9 @@ def find_errors(self, output):
             map(self._error_to_tuple, errors),
             map(self._uncovered_to_tuple,
                 coverage.get('expressions', {}).get('uncovered_locs', []),
+                repeat(set())),
+            map(self._empty_to_tuple,
+                coverage.get('expressions', {}).get('empty_locs', []),
```

Comment:
You didn't introduce this `repeat(set())` thing but 🤔 it looks wrong bc we don't iterate within '_uncovered_to_tuple' or '_empty_to_tuple'. As it stands every invocation of '_empty_to_tuple' gets a new empty 'set()'. Isn't what the original author wanted more like

```
shared_set = set()
...
    repeat(shared_set)
```

Reply:
I'm not a python guru, but I think they're equivalent? `repeat(set())` and `repeat(shared_set)` both result in an iterator that returns the same instance of a set each time. I think `_empty_to_tuple` and `_uncovered_to_tuple` should get separate sets, since they're both generating separate lint messages.

Reply:
:man_facepalming: yes sure